web/calendar.go

```diff
@@ -39,6 +39,107 @@ func getCalendarHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(xmlStr))
 }
 
+func deleteCalendarHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Some comments would help understanding this code directly,
although it is definitely understandable :)